src/Configuration.php

```diff
@@ -2,103 +2,97 @@
 namespace Wheniwork\Feedback;
 
 use Auryn\Injector;
-use Negotiation\NegotiatorInterface;
-use Spark\Resolver\ResolverInterface;
+use Equip\Configuration\ConfigurationInterface;
+use Equip\Env;
+use Wheniwork\Feedback\Service;
 
-class Configuration
+class Configuration implements ConfigurationInterface
 {
-    public function apply(Injector $injector, array $env)
-    {
-        // --------------------
-        // Initialize responders
-        // --------------------
-        $injector->delegate("\Spark\Responder\FormattedResponder", function(NegotiatorInterface $negotiator, ResolverInterface $resolver) {
-            $responder = new \Spark\Responder\FormattedResponder($negotiator, $resolver);
-            $responder = $responder->withFormatters([
-                'Spark\Formatter\JsonFormatter' => 1.0
-            ]);
-            return $responder;
-        });
+    private $env;
 
+    public function __construct(Env $env) {
+        $this->env = $env;
+    }
+
+    public function apply(Injector $injector)
+    {
         // --------------------
         // Initialize PDO
         // --------------------
-        $injector->define("\Aura\Sql\ExtendedPdo", [
-            ':dsn' => $env['DB_DSN'],
-            ':username' => $env['DB_USERNAME'],
-            ':password' => $env['DB_PASSWORD']
+        $injector->define(\Aura\Sql\ExtendedPdo::class, [
```

Comment:
I actually had trouble without it. Probably because the namespace of this file is `Wheniwork\Feedback`?